bundles/org.openhab.ui/web/src/components/widgets/system/oh-input.vue

```diff
@@ -79,6 +57,17 @@ import { OhInputDefinition } from '@/assets/definitions/widgets/system'
 
 import { useStatesStore } from '@/js/stores/useStatesStore'
 
+export function getHtmlInputType (itemType) {
```

Comment:
It somehow feels wrong having this in here and importing it in other places. I think it should rather go into a dedicated file, but the question is where to put that.

Reply:
I can move assuming we keep the feature in the item-metadata-widget. Again, if you have a suggestion for a cleaner implemnentation, it would be good.